microsoft-edge/devtools-guide-chromium/ie-mode/index.md

```diff
@@ -85,10 +85,27 @@ After you launch Microsoft Edge \(Chromium\) using a command-line argument, IE m
 
 IE mode remains unavailable until you completely close and restart Microsoft Edge \(Chromium\).  
 
+
+## Replacing IE Automation
+
+If you have an existing application that uses the [InternetExplorer][InternetExplorerObject] object to automate Internet Explorer 11, your application will not work if the Internet Explorer 11 desktop application is not available.  Internet Explorer 11 will be retired on June 15, 2022. For more information, navigate to [The future of Internet Explorer on Windows 10 is in Microsoft Edge](https://blogs.windows.com/windowsexperience/2021/05/19/the-future-of-internet-explorer-on-windows-10-is-in-microsoft-edge/).  Microsoft Edge \(Chromium\) does not support automating IE mode through the [InternetExplorer][InternetExplorerObject] object, so applications that rely on this object will need to be updated using one of the recommended alternatives below.  Applications that use the [WebBrowser][WebBrowserControl] control will continue to work and will not be affected by the removal of Internet Explorer 11.
```

Comment:
```suggestion
If you have an existing application that uses the [InternetExplorer][InternetExplorerObject] object to automate Internet Explorer 11, but the Internet Explorer 11 desktop application isn't available, your application won't work.  Internet Explorer 11 will be retired on June 15, 2022.  For more information, navigate to [The future of Internet Explorer on Windows 10 is in Microsoft Edge][BlogsWindowsExperienceFutureOfIEEdge].  Microsoft Edge \(Chromium\) doesn't support automating IE mode through the `InternetExplorer` object, so applications that rely on this object will need to be updated using one of the recommended alternatives below.  Applications that use the [WebBrowser][WebBrowserControl] control will continue to work and won't be affected by the removal of Internet Explorer 11.
```
Please test the reformatted link after the doc build, "The future of Internet Explorer on Windows 10 is in Microsoft Edge".  https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/ie-mode/?branch=pr-en-us-1336#replacing-ie-automation

Reply:
The reformatted link works.